src/bp/admin/ui/src/auth/Login.tsx

```diff
@@ -76,14 +83,14 @@ const Login: FC<Props> = props => {
   }
 
   const loadAuthConfig = async () => {
-    const { data } = await api.getAnonymous().get('/admin/auth/config')
+    const { data }: { data: AuthConfigResponse } = await api.getAnonymous().get('/admin/auth/config')
```

Comment:
You can type the data directly by passing the class to the axios call: 

`const { data } = await api.getAnonymous().get<AuthConfigResponse>('/admin/auth/config')`